assets/src/scss/base/_fonts.scss

```diff
@@ -1,5 +1,22 @@
-$font-base-size: 16px;
+@import "./tokens";
```

Comment:
Is this needed? I see that tokens is the first thing being imported in `style.scss`

Reply:
You're correct! Fixed 👍